src/main/java/uk/gov/pay/ledger/report/entity/TransactionSummaryResult.java

```diff
@@ -10,10 +10,14 @@
 
     private TransactionsStatisticsResult payments;
     private TransactionsStatisticsResult refunds;
+    private Long netIncome;
 
-    public TransactionSummaryResult(TransactionsStatisticsResult payments, TransactionsStatisticsResult refunds) {
+    public TransactionSummaryResult(TransactionsStatisticsResult payments,
+                                    TransactionsStatisticsResult refunds,
+                                    Long netIncome) {
         this.payments = payments;
         this.refunds = refunds;
+        this.netIncome = netIncome;
```

Comment:
surely this netIncome is something that the object can calculate? It doesn't have to be passed in?